### PR TITLE
Xdd 148 cosmos endpoint progress

### PR DESCRIPTION
**Pull request by mwestphall**

### CODE REVIEW

File: cosmos_service/src/make_parquet.py
Comment (mwestphall) [inline cosmos_service/src/make_parquet.py:1]: I was not able to get the Dockerfile to honor the symlink to this file that was previously included in the directory, so I just copied the file over to the src directory directly. Let me know if there's a better approach

Comment (iross) [reply cosmos_service/src/make_parquet.py:1]: I think this is fine for the moment, but I think the cleaner approach will be to move the dockerfile to `deployments/` (where the other COSMOS dockerfiles live), and then do the build out of the root directory. That way the build context would include `htcosmos/make_parquet.py`. I'll open an issue for it, but it's not a blocker on this review.

File: cosmos_service/src/app.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    job is in a not-complete state
+    """
+    with SessionLocal() as session:
+        job = session.get(CosmosSessionJob, job_id)
+        if not job:
+            raise HTTPException(status_code=404, detail="Job not found")
+        elif not job.is_completed:
+            raise HTTPException(status_code=400, detail="Job not finished")
+    output_file = f"{job.output_dir}/cosmos_output.zip"
+    return FileResponse(output_file)
```
Comment (iross) [inline cosmos_service/src/app.py:119]: Can we provide a filename in this response? I like the idea of retaining the original name of the PDF (maybe in the `CosmosSessionJob`?) and having a returned output of `{pdf_basename}_COSMOS_results.zip`

File: cosmos_service/src/scheduler.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+from db import SessionLocal
+from sqlalchemy import select, delete, ScalarResult
+from fastapi.logger import logger
+
+
+scheduler = Rocketry(config={"task_execution": "async"})
+
+
+CLEAN_JOB_FREQUENCY = "1 hour"
+SESSION_EXPIRATION = timedelta(hours = 3)
```
Comment (iross) [inline cosmos_service/src/scheduler.py:15]: Let's bump to 24 hours.

**Review by iross — COMMENTED** [review]
This is looking awesome! Just a few very minor tweaks and then I think it's good to go.

**Pull request merged**